app/customer.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: Car
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    @staticmethod
```

Comment:
Better to do as classmethod it will be named constructor
Also the name will be better load_from_dict